src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendStep.java

```diff
@@ -190,6 +203,10 @@ public Void run() throws AbortException {
                 step.setApplicationName(getBuildName());
             }
 
+            if (step.getAppVersionTagFormat() == 0) {
```

Comment:
Use a CONSTANT

Reply:
It only checks if the int variable was assigned a value, like a null check.